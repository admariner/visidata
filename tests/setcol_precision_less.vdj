#!vd -p
{"sheet": null, "col": null, "row": null, "longname": "open-file", "input": "sample_data/sample.tsv", "keystrokes": "o", "comment": null}
{"sheet": "sample", "col": "Unit_Cost", "row": "", "longname": "type-float", "input": "", "keystrokes": "%", "comment": "set type of current column to float"}
{"sheet": "sample", "col": "Unit_Cost", "row": "", "longname": "setcol-precision-less", "input": "", "comment": "show less precision in current column"}
