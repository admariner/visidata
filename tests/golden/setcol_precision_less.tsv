OrderDate	Region	Rep	Item	Units	Unit_Cost	Total
2016-01-06	East	Jones	Pencil	95	2.0	189.05
2016-01-23	Central	Kivell	Binder	50	20.0	999.50
2016-02-09	Central	Jardine	Pencil	36	5.0	179.64
2016-02-26	Central	Gill	Pen	27	20.0	539.73
2016-03-15	West	Sorvino	Pencil	56	3.0	167.44
2016-04-01	East	Jones	Binder	60	5.0	299.40
2016-04-18	Central	Andrews	Pencil	75	2.0	149.25
2016-05-05	Central	Jardine	Pencil	90	5.0	449.10
2016-05-22	West	Thompson	Pencil	32	2.0	63.68
2016-06-08	East	Jones	Binder	60	9.0	539.40
2016-06-25	Central	Morgan	Pencil	90	5.0	449.10
2016-07-12	East	Howard	Binder	29	2.0	57.71
2016-07-29	East	Parent	Binder	81	20.0	1619.19
2016-08-15	East	Jones	Pencil	35	5.0	174.65
2016-09-01	Central	Smith	Desk	2	125.0	250.00
2016-09-18	East	Jones	Pen Set	16	16.0	255.84
2016-10-05	Central	Morgan	Binder	28	9.0	251.72
2016-10-22	East	Jones	Pen	64	9.0	575.36
2016-11-08	East	Parent	Pen	15	20.0	299.85
2016-11-25	Central	Kivell	Pen Set	96	5.0	479.04
2016-12-12	Central	Smith	Pencil	67	1.3	86.43
2016-12-29	East	Parent	Pen Set	74	16.0	1183.26
2017-01-15	Central	Gill	Binder	46	9.0	413.54
2017-02-01	Central	Smith	Binder	87	15.0	1305.00
2017-02-18	East	Jones	Binder	4	5.0	19.96
2017-03-07	West	Sorvino	Binder	7	20.0	139.93
2017-03-24	Central	Jardine	Pen Set	50	5.0	249.50
2017-04-10	Central	Andrews	Pencil	66	2.0	131.34
2017-04-27	East	Howard	Pen	96	5.0	479.04
2017-05-14	Central	Gill	Pencil	53	1.3	68.37
2017-05-31	Central	Gill	Binder	80	9.0	719.20
2017-06-17	Central	Kivell	Desk	5	125.0	625.00
2017-07-04	East	Jones	Pen Set	62	5.0	309.38
2017-07-21	Central	Morgan	Pen Set	55	12.5	686.95
2017-08-07	Central	Kivell	Pen Set	42	23.9	1005.90
2017-08-24	West	Sorvino	Desk	3	275.0	825.00
2017-09-10	Central	Gill	Pencil	7	1.3	9.03
2017-09-27	West	Sorvino	Pen	76	2.0	151.24
2017-10-14	West	Thompson	Binder	57	20.0	1139.43
2017-10-31	Central	Andrews	Pencil	14	1.3	18.06
2017-11-17	Central	Jardine	Binder	11	5.0	54.89
2017-12-04	Central	Jardine	Binder	94	20.0	1879.06
2017-12-21	Central	Andrews	Binder	28	5.0	139.72
